### PR TITLE
TASK: Workspaces styling improvements

### PR DESCRIPTION
**Pull request by aertmann**
A couple of fixes, improvements and cleanup related to the workspaces module.

### CODE REVIEW

**Comment by aertmann** [comment]
:+1:

**Comment by aertmann** [comment]
image
image
image

image
image
image

File: TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Index.html [hunk 1]
```diff
@@ -56,10 +56,25 @@
...
+											</f:else>
+										</f:if>
+									</div>
 								</span>
 								<f:link.action action="show" arguments="{workspace: workspace}" class="neos-button neos-button-primary">
-									{neos:backend.translate(id: 'workspaces.review', source: 'Modules', package: 'TYPO3.Neos')}
+									<i class="icon-review"></i>
+									<f:if condition="{changesCounts.total} != 0">
+										<f:then>{neos:backend.translate(id: 'workspaces.reviewChanges', source: 'Modules', package: 'TYPO3.Neos', arguments: {0: changesCounts.total})}</f:then>
+										<f:else>{neos:backend.translate(id: 'workspaces.noChanges', source: 'Modules', package: 'TYPO3.Neos')}</f:else>
```
Comment (kdambekalns) [inline TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Index.html]: If there are no changes, I would not show the button at all.

**Comment by robertlemke** [comment]
Hey, thanks a lot for the improvements.

Got some nitpicks though:

image

I wouldn't make that button a button if there are no changes. Better just write "No changes" and make the icon inactive (light grey). Also the plural form for the "Review x change(s)" does not seem to work (should be "Review 1 change"

**Comment by robertlemke** [comment]
image

In the workspace changes view, I really wouldn't display the number of changes. Is there an added value for the editor to know the exact number of changes in that screen? I currently can't think of any.

**Comment by kdambekalns** [comment]
I agree, I'd leave the number of changes out.

**Comment by kdambekalns** [comment]
We (@robertemke, @kitsunet and me) discussed this and decided to merge this and undo the "number of changes display" bits for now.

**Comment by aertmann** [comment]
right, all good thanks

**Comment by aertmann** [comment]
was primarily the review button no leading to anything that I didn't like

**Pull request merged**